.teamcity/Gradle_Promotion/buildTypes/PromoteRelease.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package Gradle_Promotion.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2018_2.ParameterDisplay
+
+open class PromoteRelease(task: String, requiredConfirmationCode: String, branch: String = "release", init: PromoteRelease.() -> Unit = {}) : BasePromoteSnapshot(
```

Comment:
:o: `abstract` instead of `open` 

Same for all other `open` classes.

Reply:
The `open` classes are now `abstract`.